packages/mobile/src/fiatExchanges/FiatExchangeOptions.tsx

```diff
@@ -184,6 +194,24 @@ const styles = StyleSheet.create({
     height: 30,
     width: 104,
   },
+  simplexLogo: {
+    height: 59,
+    width: 111,
+  },
```

Comment:
nit: Would be nice to use a standardized height for providers logo and adjust the asset instead.

Reply:
Sounds good, will merge for now as it as we'll be updating the UI with the new flow